apps/site/next.mdx.shiki.mjs

```diff
@@ -168,6 +192,11 @@ export default function rehypeShikiji() {
       // Grabs the relevant alias/name of the language
       const languageId = codeLanguage.slice(languagePrefix.length);
 
+      // Check code for syntax errors
+      if (!checkCodeSyntax(preElementContents, languageId)) {
```

Comment:
Is this a good idea? I would rather have a separate CI process that verifies this rather than Acorn right here. Otherwise it will just dramatically slow down the build process.